content/pt/docs/getting-started/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+título: Primeiros Passos
+descrição: Comece a utilizar o OpenTelemetry com base na sua função.
```

Comment:
```suggestion
título: Primeiros Passos
descrição: Comece a utilizar o OpenTelemetry com base na sua função.
```

FYI, do not translate they keys here, this will break the page build + linters.

I will apply a fix commit and then it should look good.